pkg/volume/volume_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	"testing"
 )
 
-const supportedCasTypeCount = 3
+const supportedCasTypeCount = 4
```

Comment:
I'm not sure why I had to bump this, I didn't add any method in the casMaps

Reply:
Its because the PR for zfs localpv listing has been merged. So the supported cas-types have been increased to 4.